### DISCUSSION TITLE
Incorrect Last Commit Date Display in Repository

### DISCUSSION THREAD

**Discussion by Ruminini** [community/community#126767] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I’ve noticed an issue with the commit date displayed on repositories.
Today is 2024-05-31, but the "Last commit date" column shows "4 days ago".
Upon hovering over this date, it shows the actual commit date as 2024-04-27, so the difference is **1 month** and 4 days.

![image](https://github.com/community/community/assets/63514002/9aaefcca-5a4b-4ebd-9967-c58e2df3fd6e)

It doesn't seem to matter which browser I use.

Thinking aloud, I wonder if this issue might be related to the fact that today is May 31, and perhaps the server is located in a region where it is already June 1.

**Comment by ebndev** [comment]
Hi @Ruminini, thanks for flagging this!

I'll keep a log of this issue as a few other users have mentioned this as well. The GitHub product team will evaluate the feedback but will not be able to respond to every submission. You can keep an eye on our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
